.github/workflows/ci-documentbot.yml

```diff
@@ -27,6 +27,10 @@ on:
       - labeled
       - unlabeled
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
are you sure that in progress builds should be cancelled? That could lead to some events being skipped. it  that fine?

Reply:
Yes, my previous test (without `cancel-in-progress`) at https://github.com/open-github/pulsar/pull/6 shows we have to cancel it, or an inconsistent state will occur.

Reply:
ok